### PR TITLE
BTC Warning banner when transactions are pending

### PR DESCRIPTION
**Pull request by hedi-edelbloute**
### 📝 Description

Add a warning banner when atleast one transaction is pending for BTC accounts

### ❓ Context

- **Impacted projects**: `ledger-live-desktop` 
- **Linked resource(s)**: `LIVE-2020 , LIVE-3398` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

image
image

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/components/OperationsList/AmountCell.jsx [hunk 1]
```diff
@@ -68,7 +68,13 @@ class AmountCell extends PureComponent<Props> {
                   showCode
                   fontSize={4}
                   alwaysShowSign
-                  color={amount.isNegative() ? "palette.text.shade80" : undefined}
+                  color={
```
Comment (hzheng-ledger) [inline apps/ledger-live-desktop/src/renderer/components/OperationsList/AmountCell.jsx:71]: Would it be more concise if
```
operation.type === "OUT"? "palette.text.shade80" : isConfirmed ? undefined : colors.warning
```
?

File: apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.jsx [hunk 2]
```diff
@@ -302,7 +303,13 @@ const OperationD: React$ComponentType<Props> = (props: Props) => {
                   }
                 >
                   <FormattedVal
-                    color={amount.isNegative() ? "palette.text.shade80" : undefined}
+                    color={
+                      !isConfirmed && operation.type === "IN"
```
Comment (hzheng-ledger) [inline apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.jsx:307]: same as my above comment

File: apps/ledger-live-desktop/src/renderer/families/bitcoin/SendRecipientFields.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import { getMainAccount } from "@ledgerhq/live-common/account/index";
+import { PendingOperation } from "@ledgerhq/errors";
+
+class SendRecipientFields extends PureComponent {
+  render() {
+    const { confirmationsNb, account } = this.props;
+    const pendingOperationError = new PendingOperation();
+    const operations = account.pendingOperations.concat(account.operations);
+    const incomingTransactionPending =
+      operations.find(
```
Comment (hzheng-ledger) [inline apps/ledger-live-desktop/src/renderer/families/bitcoin/SendRecipientFields.jsx:17]: use "some" method instead of "find" would make the code clearer

**Review by hzheng-ledger — APPROVED** [review]
It looks good to me. Have a look at my comment if you have time. Only some minor issues. Good job

File: apps/ledger-live-desktop/src/renderer/components/OperationsList/AmountCell.jsx (hunk 1 above)
Comment (hedi-edelbloute) [reply apps/ledger-live-desktop/src/renderer/components/OperationsList/AmountCell.jsx:71]: I understand what you want to do but i think it's better to separate the case "not confirmed and incoming", discussed on slacked

File: apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.jsx (hunk 2 above)
Comment (hedi-edelbloute) [reply apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.jsx:307]: see above

File: apps/ledger-live-desktop/src/renderer/families/bitcoin/SendRecipientFields.jsx (hunk 3 above)
Comment (hedi-edelbloute) [reply apps/ledger-live-desktop/src/renderer/families/bitcoin/SendRecipientFields.jsx]: Done, thanks !

**Comment by desirendr** [comment]
@hedi-edelbloute merging this as it's been validated by @mdeidda-ledger and reviewed !

**Pull request merged**